src/ExpandObjects/epfilter.f90

```diff
@@ -17823,6 +17824,7 @@ SUBROUTINE CreateNewUnitary
     CALL WriteComment('21.11C (70F) entering air dry-bulb temperature and 21.11C (70F) entering water temperature')
     CALL CreateNewObj('Coil:Heating:WaterToAirHeatPump:EquationFit')
     CALL AddToObjFld('Name', base + ussAirHandlerNameOff,' Heating Coil')
+    CALL AddToObjStr('Availability Schedule Name','')
```

Comment:
Adds a blank availability schedule name field when the `ExpandObjects` program adds the DX coils.